projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -22,24 +22,24 @@
   - projects/control-service/projects/helm_charts/pipelines-control-service/**/*
 
 .control_service_base_build:
-  image: docker:19.03.15
+  image: docker:20.10.22
```

Comment:
nit: I am not sure if you feel like it. But you can extract this as "variable" (using yaml anchors or gitlab reference syntaxes)

```
.images:docker:
  image: docker:20.10.22
```

Reply:
there is very little replication now that I use the extend functionality for the dind. So I won't make this change.